File: cristopher/Problema0376/problema0376.js
const prompt = require("prompt-sync")({ sigvar: true });

var matrizA = [[0, 0, 0], [0, 0, 0]]
var total = 0

for(var i = 0; i < 2; i++){
    for(var j = 0; j < 3; j++){
        matrizA[i][j] = prompt("Digite o número na posição ["+i+", "+j+"]: ")
        if(!(matrizA[i][j] >= 5 && matrizA[i][j] <= 15)){
            total++
        }
    }
}

console.log(total)

File: cristopher/Problema0376/problema0376.py
matrizA = [[0, 0, 0], [0, 0, 0]]
total = 0

for l in range (0, 2):
    for c in range (0, 3):
        matrizA[l][c] = int(input(f"Digite um valor na posição [{l, c}]: "))
        if(not(matrizA[l][c] >= 5 and matrizA[l][c] <= 15)):
            total = total + 1

print(matrizA)
print(total)
